<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Jim Morris</author>
 <class>ZoomForm</class>
 <widget class="QWidget" name="ZoomForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>305</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="zoom1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="zoom2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="zoom3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="zoom4">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="zoom5">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="zoom6">
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="zoom7">
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="zoom8">
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="zoom9">
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="zoom10">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="zoom11">
     <property name="text">
      <string>11</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="zoom12">
     <property name="text">
      <string>12</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="zoom13">
     <property name="text">
      <string>13</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="zoom14">
     <property name="text">
      <string>14</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="zoom15">
     <property name="text">
      <string>15</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="zoom16">
     <property name="text">
      <string>16</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="zoom17">
     <property name="text">
      <string>17</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="zoom18">
     <property name="text">
      <string>18</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="zoom19">
     <property name="text">
      <string>19</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="zoom20">
     <property name="text">
      <string>20</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
